README.md

```diff
@@ -9,12 +9,6 @@
 [![crates.io](https://img.shields.io/crates/v/fzf-make?style=flatflat-square)](https://crates.io/crates/fzf-make)
 [![Crates.io Total Downloads](https://img.shields.io/crates/d/fzf-make)](https://crates.io/crates/fzf-make)
 
-<p align="center">
-    [English]
-    [<a href="doc/README-de.md">Deutsch</a>]
-    [<a href="doc/README-fr.md">Français</a>]
-</p>
-
```

Comment:
I'm sorry for giving you a misunderstanding.
Please revert this deletion. 🙏

In description of https://github.com/kyu08/fzf-make/issues/301, I wanted to show that READMEs other than English just  need not to modified.

I'm going to delete them by another PR.

Reply:
fixed